docs/components/IconSearch.tsx

```diff
@@ -39,7 +36,7 @@ const Item: React.FC<{ name: string }> = ({ name }) => {
         >
           {/* eslint-disable-next-line */}
           {/*  @ts-ignore */}
-          {React.createElement(evergreen[name] as any, {
+          {React.createElement(evergreen[name] as React.ReactElement, {
```

Comment:
do you need the @ts-ignore now that you fixed the any?

Reply:
Good catch - probably not!